C-Plus-Plus/computational_geometry/Check_Orientation_of_three_points.cpp

```diff
@@ -0,0 +1,76 @@
+/* This Program is to check the Orientation of three given points on a 2-D plane
+     i.e, a) Counter-Clockwise
+          b) Clockwise
+          c) Collinear
+ Based on the slope, we can categorize the orientation of the three points.
+ If the slope of line segment (a,b)> slope of line segment (b,c), then the orientation is clockwise or right turn
+ and vice versa for counter - clockwise or left turn.
+ When the three points are collinear, the slope of both the lines are equal*/
+
+
+
```

Comment:
Remove extra lines of Code

Reply:
@HarshCasper Removed the extra lines. Is the code fine now?